[2022-08-23 11:58:53,267] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-21T13:00:00+00:00 [queued]>
[2022-08-23 11:58:53,285] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-21T13:00:00+00:00 [queued]>
[2022-08-23 11:58:53,285] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:58:53,286] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:58:53,287] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:58:53,326] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-10-21 13:00:00+00:00
[2022-08-23 11:58:53,335] {standard_task_runner.py:52} INFO - Started process 308 to run task
[2022-08-23 11:58:53,341] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-10-21T13:00:00+00:00', '--job-id', '2471', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmp5l5mmze0', '--error-file', '/tmp/tmprzzlb3dp']
[2022-08-23 11:58:53,342] {standard_task_runner.py:80} INFO - Job 2471: Subtask print_world
[2022-08-23 11:58:53,460] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-21T13:00:00+00:00 [running]> on host 9f4b06248101
[2022-08-23 11:58:53,556] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-10-21T13:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-21T13:00:00+00:00
[2022-08-23 11:58:53,557] {logging_mixin.py:115} INFO - world
[2022-08-23 11:58:53,557] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:58:53,586] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20211021T130000, start_date=20220823T115853, end_date=20220823T115853
[2022-08-23 11:58:53,632] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:58:53,689] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
